app/services/action_queue.rb

```diff
@@ -172,14 +165,6 @@ def fake_card_info
     def get_payment_account
       "GoCardless #{data[:currency]}"
     end
-
-    def action_fields
-      {
-        recurring_id:      @action.member_id,
```

Comment:
Why are we removing this data? I don't see these fields moving anywhere, they're just gone.